src/plugins/recordTypes/location/fields.js

```diff
@@ -0,0 +1,27 @@
+export default (configContext) => {
+  const {
+    TextInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  return {
+    document: {
+      'ns2:locations_common': {
+        locTermGroupList: {
+          locTermGroup: {
+            termSource: {
+              [config]: {
+                view: {
+                  type: TextInput,
```

Comment:
Need `props: null` here -- otherwise you'll see an error in the console about invalid props being passed to TextInput, because the `source` prop will still be inherited from the default config.